tools/azure-sdk-tools/setup.py

```diff
@@ -10,6 +10,7 @@
     "wheel",
     "Jinja2",
     "MarkupSafe==2.0.1",
+    # black,
```

Comment:
this line is surplus

Reply:
> this line is surplus

Add notes here as a reminder, which is more standardized